packages/lab/src/dropdown-next/DropdownNext.tsx

```diff
@@ -205,8 +205,8 @@ export const DropdownNext = forwardRef(function DropdownNext(
         role="combobox"
         aria-haspopup="listbox"
         aria-expanded={open}
-        aria-activedescendant={disabled ? undefined : activeDescendant}
-        tabIndex={disabled ? -1 : 0}
+        aria-activedescendant={activeDescendant}
```

Comment:
Can we add tests for these 2 line of changes.

Reply:
@origami-z  updated cypress